.cog/package_templates/go/docs/How-To/converting-a-dashboard.md

```diff
@@ -0,0 +1,31 @@
+# Converting a dashboard
+
+```go
+package main
+
+import (
+    "encoding/json"
+    "fmt"
+
+    "github.com/grafana/grafana-foundation-sdk/go/cog/plugins"
```

Comment:
Is this suppose to point towards a branch? Trying this out I get an error:

```
no required module provides package github.com/grafana/grafana-foundation-sdk/go/cog/plugins; to add it:
	go get github.com/grafana/grafana-foundation-sdk/go/cog/plugins
```

Same with `github.com/grafana/grafana-foundation-sdk/go/dashboard`

Reply:
Oh never mind running `go get github.com/grafana/grafana-foundation-sdk/go@next+cog-v0.0.x` fixes this.